src/actions/auth.js

```diff
@@ -0,0 +1,69 @@
+import firebase from 'firebase';
+
+const firebaseApp = {
+  apiKey: 'AIzaSyDwnsJcOAy2cP2lyTFdWcanOMTf2DRZpIs',
+  authDomain: 'local-cb336.firebaseapp.com',
+  databaseURL: 'https://local-cb336.firebaseio.com',
+  storageBucket: 'local-cb336.appspot.com',
+  messagingSenderId: '514582937003'
+};
+
+firebase.initializeApp(firebaseApp);
+
+const provider = new firebase.auth.GoogleAuthProvider();
+
+export const startListeningToAuth = () => {
+  return (dispatch, getState) => {
+    firebase.auth().onAuthStateChanged(user => {
+      if (user) {
```

Comment:
It's giving me this error, and not logging anything for user. 
mapStateToProps() in Connect(SignIn) must return a plain object. Instead received undefined.

Uncaught Error: Actions must be plain objects. Use custom middleware for async actions.(…)